### PR TITLE
353 router 4 bootstrap router

### PR DESCRIPTION
**Pull request by t-laird**

### CODE REVIEW

**Comment by t-laird** [comment]
As discussed, this ended up being a fix to use React-Router links where there were previously bootstrap hrefs instead. This also necessitated a package downgrade to the previous version of react-router-bootstrap.

File: client/src/components/CreateCampaignStep2.js [hunk 1]
```diff
@@ -40,9 +40,7 @@ class CreateCampaignStep2 extends React.Component {
 
   setOptionalInfo = async e => {
     e.preventDefault();
-    const {
-      target: { name, email, phone, address, buildingCount, unitCount }
-    } = e;
+    const { target: { name, email, phone, address, buildingCount, unitCount } } = e;
```
Comment (katiescruggs) [inline client/src/components/CreateCampaignStep2.js]: Totally did not know you could do nested destructuring like this -- cool.

File: client/src/stylesheets/components/_navbar.scss [hunk 2]
```diff
@@ -55,13 +55,9 @@
   background-color: $dark-blue;
 }
 
-.dropdown.open {
-  background-color: magenta;
-}
-
```
Comment (katiescruggs) [inline client/src/stylesheets/components/_navbar.scss]: Thanks for deleting my magentas!

**Comment by JasminBarco** [comment]
we think this is done and merged...

**Pull request closed without merging**